portality/scripts/generate_docs_index.py

```diff
@@ -3,13 +3,17 @@
 """
 
 import os
+from datetime import datetime
 
 BRANCH_PREFIXES = [
     "feature",
     "hotfix",
     "release"
 ]
 
+IGNORE = [
```

Comment:
other things to ignore? cache, localstore, secrets etc. But I guess those aren't code

Reply:
It's just the files to ignore in the doaj-docs directory that shouldn't be indexed, which is just the `.git` directory for now.